### DISCUSSION TITLE
"rustdesk --silent-install" does not exit after completing

### DISCUSSION THREAD

**Discussion by yaroz** [rustdesk/rustdesk#8957] [General]
### Bug Description

When running rustdesk executable with --silent-install, it does not exit after completing the install.  This is on the windows .exe file

### How to Reproduce

in powershell running the following command will not exit and will keep the process open.

$rusttempfile = "C:\temp\rustdesk.exe"
Start-Process $rusttempfile -ArgumentList "--silent-install" -wait

### Expected Behavior

Exit with an errorcode indicating if it was successful or not

### Operating system(s) on local side and remote side

Windows 10/11

### RustDesk Version(s) on local side and remote side

1.3.0

### Screenshots

na

**Comment by GuillaumeHullin** [comment]
Same here using Chocolatey package
